web/public/js/controllers/accept-session-invite-controller.js

```diff
@@ -29,8 +29,9 @@
         $state.go('dojo-list');
       }
     }, function (err) {
-      if(err) {
-        console.error(err);
+      if (err.status === 410) {
+        alertService.showError($translate.instant('Full up'));
```

Comment:
Definitely needs a more detailed error.

`This event is fully booked. As a result, you can no longer accept your invitation.`?

/cc @conoramurphy

Reply:
Yeha, reused the one used for the event booking to avoid a new translation string